GUI-dev-old/package.json

```diff
@@ -0,0 +1,163 @@
+    "@angular/router": "3.3.0",
+    "@angularclass/conventions-loader": "^1.0.2",
+    "@angularclass/hmr": "~1.2.0",
+    "@angularclass/hmr-loader": "~3.0.2",
+    "amcharts3": "https://github.com/amcharts/amcharts3",
+    "ammap3": "https://github.com/amcharts/ammap3",
+    "angular2-jwt": "^0.1.26",
+    "animate.css": "^3.5.1",
+    "assets-webpack-plugin": "^3.4.0",
+    "bootstrap": "4.0.0-alpha.4",
+    "bootstrap-loader": "2.0.0-beta.12",
+    "chart.js": "^1.1.1",
+    "chartist": "^0.9.7",
+    "ckeditor": "^4.5.9",
+    "core-js": "^2.4.1",
+    "easy-pie-chart": "^2.1.7",
+    "extract-text-webpack-plugin": "2.0.0-beta.3",
+    "font-awesome": "^4.6.3",
+    "font-awesome-sass-loader": "^1.0.2",
+    "formBuilder": "^1.24.3",
+    "http-server": "^0.9.0",
+    "ie-shim": "^0.1.0",
+    "ionicons": "^2.0.1",
+    "is-electron-renderer": "^2.0.0",
+    "jquery": "^2.2.3",
+    "jquery-slimscroll": "^1.3.6",
+    "jquery-ui-bundle": "^1.11.4",
+    "lodash": "^4.12.0",
+    "ng2-bootstrap": "1.1.14",
+    "ng2-ckeditor": "1.0.7",
+    "ng2-smart-table": "0.4.0-5",
+    "ng2-tree": "^0.0.2-7",
+    "ng2-uploader": "1.1.0",
+    "normalize.css": "^4.1.1",
+    "postcss-loader": "^1.0.0",
+    "rxjs": "5.0.0-rc.4",
+    "tether": "^1.2.4",
+    "zone.js": "^0.7.2"
+  },
+  "devDependencies": {
+    "angular2-template-loader": "^0.5.0",
+    "@types/electron": "^1.3.20",
+    "@types/hammerjs": "^2.0.33",
+    "@types/jasmine": "^2.2.34",
+    "@types/node": "^6.0.38",
+    "@types/source-map": "^0.1.27",
+    "@types/uglify-js": "^2.0.27",
+    "@types/webpack": "^1.12.34",
+    "@types/lodash": "0.0.28",
+    "gh-pages": "^0.11.0",
+    "parse5": "^2.2.2",
+    "rimraf": "^2.5.2",
+    "script-ext-html-webpack-plugin": "^1.3.2",
+    "codelyzer": "2.0.0-beta.1",
+    "tslint": "~4.0.0",
+    "ts-helpers": "1.1.2",
+    "ts-node": "^1.3.0",
+    "typedoc": "^0.5.0",
+    "typescript": "2.0.3",
+    "awesome-typescript-loader": "^2.2.1",
+    "tslint-loader": "^2.1.3",
+    "url-loader": "^0.5.7",
+    "style-loader": "^0.13.1",
+    "raw-loader": "0.5.1",
+    "source-map-loader": "^0.1.5",
+    "string-replace-loader": "1.0.5",
+    "imports-loader": "^0.6.5",
+    "json-loader": "^0.5.4",
+    "css-loader": "^0.25.0",
+    "exports-loader": "^0.6.3",
+    "expose-loader": "^0.7.1",
+    "file-loader": "^0.9.0",
+    "to-string-loader": "^1.1.4",
+    "sass-loader": "^4.0.2",
+    "resolve-url-loader": "^1.4.3",
+    "node-sass": "^3.5.3",
+    "html-webpack-plugin": "^2.21.0",
+    "copy-webpack-plugin": "^3.0.1",
+    "webpack": "2.1.0-beta.25",
+    "webpack-dashboard": "^0.1.8",
+    "webpack-dev-middleware": "^1.6.1",
+    "webpack-dev-server": "^2.1.0-beta.9",
+    "webpack-md5-hash": "^0.0.5",
+    "webpack-merge": "^0.15.0",
+    "compression-webpack-plugin": "^0.3.1",
+    "es6-promise": "^3.1.2",
+    "es6-shim": "^0.35.0",
+    "es7-reflect-metadata": "^1.6.0",
+    "electron": "^1.4.0"
```

Comment:
Do we need this?
Please check starter-kit etc. if we need this package.
In general, check the whole `package.json` again and remove scripts and npm modules if they are not needed.

Reply:
GUI-dev-old is just the old GUI archived in order to make room for the new ngx-admin. The content of the old GUI was not altered as part of this issue.

Reply:
never mind this was my fault...wrong folder 😆